src/decoder/Makefile

```diff
@@ -16,4 +24,20 @@ ADDLIBS = ../lat/kaldi-lat.a ../hmm/kaldi-hmm.a \
           ../tree/kaldi-tree.a ../util/kaldi-util.a \
           ../matrix/kaldi-matrix.a ../base/kaldi-base.a
 
+cuda-decoder_test.o: cuda-decoder_test.cu
+	$(CUDATKDIR)/bin/nvcc  -G -g -O0 -v -c $(CUDA_ARCH) cuda-decoder_test.cu 
+
+cuda-decoder_test:cuda-decoder_test.o
+	g++ -g -O0 -o cuda-decoder_test $(CUDA_LDFLAGS)  $(CUDA_LDLIBS)  -lcuda cuda-decoder_test.o
+
+cuda-decoder_prelink.o: cuda-decoder.cu cuda-decoder.h cuda-decoder-utils.h
```

Comment:
is there a way to split this into multiple lines? It's too long

Reply:
done